content/src/arrows.js

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////////////////////////////////////////////////
+// Arrow Support to Create Code-flow Arrows.
+///////////////////////////////////////////////////////////////////////////
+var $              = require('jquery');
+
+
+function curvedVertical(x1, y1, x2, y2) {
+  var radius = Math.abs(y1 - y2);
+  var line = [];
+  y2 = y2 - 12;
+  
+  x1 = parseFloat(x1);
+  y1 = parseFloat(y1);
+  x2 = parseFloat(x2);
+  y2 = parseFloat(y2);
+  return 'M'+ x1 + "," + y1 + " " + 'A'+ radius + "," + radius + " 1 0,1 " + x2 + "," + y2;
+};
+
+function arrow(show_fade, startcoords, endcoords, x_val, offset_left, offset_top){
```

Comment:
Could arrow() be renamed into something more verb-like? like drawArrow or something? that way it'd be easier to grep/search for in the rest of the code, too (otherwise there's also an arrow data structure and another related arrow function that get filtered in)